test/test_browser.py

```diff
@@ -5910,14 +5963,16 @@ def setUp(self):
     self.set_setting('INITIAL_MEMORY', '4200mb')
     self.set_setting('GLOBAL_BASE', '4gb')
     self.emcc_args.append('-Wno-experimental')
+    # Without this we get a warning about GLOBAL_BASE being ignored when used with SIDE_MODULE
+    self.emcc_args.append('-Wno-unused-command-line-argument')
     self.require_wasm64()
 
 
 class browser64_2gb(browser):
   def setUp(self):
     super().setUp()
     self.set_setting('MEMORY64')
-    self.set_setting('INITIAL_MEMORY', '2200gb')
```

Comment:
Wow this was trying to use a lot of memory before :smile: how did it not error?

Reply:
We don't run any tests in this mode.  We normally use the `browser64_4gb` mode.. this mode was added for local testing